docs/amber_script.md

```diff
@@ -270,6 +270,11 @@ contain image attachment content, depth/stencil content, uniform buffers, etc.
   # be a `FORMAT` buffer.
   BIND BUFFER {buffer_name} AS sampled_image
 
+  # Attach |buffer_name| as a combined image image sampler. The provided
+  # buffer must be a `FORMAT` buffer. A sampler |sampler_name| must also
```

Comment:
I don't think this needs to restrict to FORMAT buffers. We can assemble a format name for things like a vec2 or vec3. You'll just end up with R32G32B32_SFLOAT. If you want something more specific, you can use FORMAT thought.

Reply:
Done